kotlinx.fuzz.gradle/build.gradle.kts

```diff
@@ -1,8 +1,11 @@
 plugins {
     id("org.plan.research.kotlinx-fuzz-module")
     `kotlin-dsl`
+    id("com.gradle.plugin-publish") version "1.2.1"
 }
 
+group = "org.plan.research" // TODO: org.jetbrains.research(.plan) ?
```

Comment:
1. I am getting `java.lang.NullPointerException: group.displayName must not be null` when building. It doesn't affect anything but it is a little bit annoying, so if you could fix it, it would be wonderful
2. We have `GROUP_ID` constant in `versions.kt` you may consider looking at it.
3. We have `group` set in the `org.plan.research.kotlinx-fuzz-module.gradle.kts` why should we change it here?

Reply:
> I am getting java.lang.NullPointerException

I'm not, neither from IDEA nor console, with a clean build 👀 Just rebased on the latest `main`, maybe you can try again?

> We have group set in the org.plan.research.kotlinx-fuzz-module.gradle.kts

Missed that 🙈 removed my redundant line